crawler/peering.go

```diff
@@ -114,7 +112,10 @@ func (p *Peering) runOrcherster() {
 				if ok {
 					continue
 				}
-				p.dialC <- nextNode.hostInfo
+				isTimeToDial := nextNode.nextDialTime.Before(time.Now())
+				if isTimeToDial {
+					p.dialC <- nextNode.hostInfo
+				}
 				dialedCache[nextNode.hostInfo.ID] = struct{}{}
 			} else {
 				// still check the contexts in case we have to interrupt
```

Comment:
What is supposed to happen in the case `<-startT.C`

Reply:
I believe that is a delay for when there are no nodes initially, to not loop constantly.

Reply:
Cool :) That makes sense, thanks for the explanation